data/competencies.yml

```diff
@@ -64,6 +64,7 @@
   examples:
     - Fixes vulnerabilities in dependencies raised by Snyk
     - Fixes vulnerabilities raised by the cyber security team
+    - Doesn't introduce vulnerabilities outlined in the OWASP top 10
```

Comment:
This example doesn't mention anything about fixing a security vulnerability, which is what I think this competency is for. Should the summary section be changed to reflect that this isn't only about fixing vulnerabilities?

Reply:
I'm wondering what OWASP stands for.

Reply:
@JakeChampion good point.
@j-mes I can expand that, it's the Open Web Application Security Project, commonly referred to as OWASP, however since you're going to have to google that anyway I'm not sure how important it is to expand it - you just have more words to type into the search box now.

Reply:
Googling now :+1: I thought it was a special FT acronym…